modules/renter/contractor/contractor.go

```diff
@@ -84,6 +84,12 @@ func (c *Contractor) Contract(hostAddr modules.NetAddress) (modules.RenterContra
 	return modules.RenterContract{}, false
 }
 
+// ContractByID returns the contract with the id specified, if it exists.
+func (c *Contractor) ContractByID(id types.FileContractID) (modules.RenterContract, bool) {
+	contract, exists := c.contracts[id]
+	return contract, exists
```

Comment:
shouldn't this be causing race conditions? It's an exported function with no threading safety